opcua/common/manage_nodes.py

```diff
@@ -85,14 +77,48 @@ def create_variable(parent, *args):
     args are nodeid, browsename, value, [variant type], [data type]
     or idx, name, value, [variant type], [data type]
     """
-    nodeid, qname = _parse_add_args(*args[:2])
+    nodeid, qname = _parse_nodeid_qname(*args[:2])
     val, datatype = _to_variant_with_datatype(*args[2:])
     if datatype and not isinstance(datatype, ua.NodeId):
         raise RuntimeError()
 
     return node.Node(parent.server, _create_variable(parent.server, parent.nodeid, nodeid, qname, val, datatype, isproperty=False))
 
 
+def create_variable_type(parent, nodeid, bname, datatype):
```

Comment:
In this was clients can add types wherever they want. Is this OK solution or should we add some additional checking that the NodeId is descendant of BaseType?